code/modules/client/loadout/loadout_footwear.dm

```diff
@@ -14,6 +14,34 @@
 	display_name = "wooden sandals"
 	path = /obj/item/clothing/shoes/sandal
 
+/datum/gear/footwear/work_boots
+	display_name = "work boots"
+	path = /obj/item/clothing/shoes/workboots
+
+/datum/gear/footwear/laceup
+	display_name = "laceup shoes"
+	path = /obj/item/clothing/shoes/laceup
+
+/datum/gear/footwear/jackboots
+	display_name = "jackboots"
+	description = "The premire choice of oppressors for 50 years!"
```

Comment:
Misspelling
```suggestion
	description = "The premier choice of oppressors for 50 years!"
```

Reply:
I also meant to write "Over" 50 years, because I think jackboots have been around for far longer than that, just in case you happen to see this Oricana.